Dockerfile

```diff
@@ -163,6 +165,9 @@ COPY --from=erd-builder /app/erd-go /usr/local/bin/
 # for backward compatibility
 RUN ln -snf /usr/local/bin/erd-go /usr/local/bin/erd
 
+# Fixes an issue with 2 nokogiri versions breaking asciidoctor-epub3 on arm64
+RUN if [[ ${TARGETARCH} == arm64 ]]; then gem uninstall nokogiri -v '1.16.0'; fi
```

Comment:
What about nokogiri?

Reply:
There are, in fact, 2 installations of `nokogiri` in the case of ARM Alpine, hence the uninstallation of this one